packages/wrangler/src/event-subscriptions/products.ts

```diff
@@ -0,0 +1,112 @@
+import assert from "node:assert";
+import { CommandLineArgsError } from "../errors";
+
+export type EventSubscription = {
+	id: string;
+	created_at: Date;
+	modified_at: Date;
+	name: string;
+	enabled: boolean;
+	source:
+		| { service: "slurper" }
+		| { service: "workflows"; workflow_name: string }
+		| { service: "workersAI"; model: string }
+		| { service: "workersBuilds"; script: string };
+	destination: { service: "queues"; queue_id: string };
+	events: string[];
+};
+
+export const products = (
+	_accountTag: string
+): Record<EventHubProductId, EventHubProductSpec<EventHubSource>> => ({
+	slurper: {
+		validate: (source) => {
+			assert(source === "slurper");
+			return { service: "slurper" };
+		},
+		format: ({ service }) => {
+			assert(service === "slurper");
+			return service;
+		},
+	},
+	workersAI: {
+		validate: (source) => {
+			const [service, model] = source.split(".");
+			assert(service === "workersAI");
+
+			if (!model) {
+				throw new CommandLineArgsError(
+					`Invalid source. Must be formatted as workersAI.<model>`
+				);
+			}
+
+			return { service, model };
+		},
```

Comment:
I'm going to ask all products that onboard to throw `CommandLineArgsError` for invalid inputs, which should let us show nice errors.